cmd/awscollector/main.go

```diff
@@ -27,7 +28,6 @@ import (
 	"github.com/aws-observability/aws-otel-collector/tools/version"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/service"
-	"go.uber.org/zap"
```

Comment:
Restore this.  It doesn't belong with the stdlib imports.

Reply:
sure, thanks!